python/tests/udf/keras_sql_udf_test.py

```diff
@@ -60,10 +60,8 @@ def test_simple_keras_udf(self):
         with IsolatedSession(using_keras=True) as issn:
             model = Sequential()
             model.add(Flatten(input_shape=(640, 480, 3)))
-            model.add(Dense(units=64))
-            model.add(Activation('relu'))
-            model.add(Dense(units=10))
-            model.add(Activation('softmax'))
+            # model.add(Dense(64, activation='relu'))
+            model.add(Dense(16, activation='softmax'))
```

Comment:
can we leave a comment here suggesting why we removed these lines?

Reply:
Made one comments to explain it.